tests/spanner_dbapi/test_cursor.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+"""Cursor() class unit tests."""
+
+import unittest
+from unittest import mock
+
+from spanner_dbapi import connect, InterfaceError
+
+
+class TestCursor(unittest.TestCase):
+    def test_close(self):
+        with mock.patch(
+            "google.cloud.spanner_v1.instance.Instance.exists", return_value=True
+        ):
+            with mock.patch(
+                "google.cloud.spanner_v1.database.Database.exists", return_value=True
+            ):
+                connection = connect("test-instance", "test-database")
+
+        cursor = connection.cursor()
+        cursor.close()
+
+        self.assertTrue(cursor.is_closed)
```

Comment:
Here and below, might be worth adding an `assertFalse` check for the same thing before calling `close`.

Reply:
Done